_test/test/goldens/generated_build_script.dart

```diff
@@ -108,23 +101,30 @@ final _builders = <_i1.BuilderApplication>[
       }),
       defaultReleaseOptions: _i8.BuilderOptions({'compiler': 'dart2js'}),
       appliesBuilders: ['build_web_compilers:dart2js_archive_extractor']),
+  _i1.apply('build_web_compilers:sdk_js_copy', [_i7.sdkJsCopyBuilder],
```

Comment:
We might want to add a `runs_before: [build_web_compilers|ddc]` to this one...

Reply:
Sorry that would be `build_web_compilers|entrypoint`.

Reply:
https://github.com/dart-lang/build/pull/2490